diagnostics/bootnodes.go

```diff
@@ -0,0 +1,25 @@
+package diagnostics
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/ledgerwatch/erigon/turbo/node"
+)
+
+func SetupBootnodesAccess(metricsMux *http.ServeMux, node *node.ErigonNode) {
+	metricsMux.HandleFunc("/bootnodes", func(w http.ResponseWriter, r *http.Request) {
+		w.Header().Set("Access-Control-Allow-Origin", "*")
+		w.Header().Set("Content-Type", "application/json")
+
+		bootnodes := node.Node().Config().P2P.BootstrapNodesV5
```

Comment:
i think need add nil-check of `p2p` object, because in future - internalcl will handle p2p and erigon will have no p2p.